Library/Homebrew/cmd/--prefix.rb

```diff
@@ -20,18 +20,67 @@ def __prefix_args
         If <formula> is provided, display the location in the Cellar where <formula>
```

Comment:
On lines 17-18 it'd be nice to use https://github.com/Homebrew/brew/blob/486e76c1f2bc8e350be26000be9f7d770efa9a30/Library/Homebrew/global.rb#L71-L73